skt/__init__.py

```diff
@@ -409,10 +409,16 @@ def merge_patch_file(self, path):
             raise Exception("Patch %s not found" % os.path.abspath(path))
         abspath = os.path.abspath(path)
         try:
-            self.git_cmd("am", abspath)
-        except subprocess.CalledProcessError:
+            subprocess.check_output(["git", "am", abspath],
+                                    cwd=self.wdir,
+                                    env=dict(os.environ, **{'LC_ALL': 'C'}))
```

Comment:
It would be nice to unify those somehow eventually, so we don't have to copy-paste things around.

Reply:
Indeed, eventually factoring out the common code would be nice. Or even modifying `git_cmd` to be able to choose `check_call` or `check_output`. But that's for a different discussion

Reply:
@spbnick I see it is good idea to unify those parameter. So, I create another PR to do it: #117